_data/Aquahamzah/Aquahamzah.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Aquahamzah
+institution: City, University of London
+profile_pic: aquahamzah.jpg
+quote: His StackOverflow requests...it's over 9000!
+github_user: aquahamzah
+---
+
```

Comment:
Can you please remove the extra new line?

Reply:
Hi there, just wondering if that shows up correctly for you now? Not entirely sure why it had the extra line! Apologies.